### DISCUSSION TITLE
Another unique insight on the CMB and physics of rotation

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#279] [Astrophysics and Physics]
So I have already extensively written original ideas or knowledge about different aspects of the CMB. 
e.g. here: https://github.com/orgs/a-cosmology-group/discussions/275
Since I have read a record high number of papers on the topic, I am able to derive insights and connect distant topics in order to make significant advances in cosmology.

Notably, among my main contributions on the topic yet are:
1) proving that the CMB cannot be nearby nor can it be recent (despite those ideas having some merit, there is still the possibility of nearby interactions though). Moreover it very sadly makes the case for an homeostatic universe much harder to defend.
2) arguing for CMB as generated by cosmic dust, as proven by the missing dust opacification signature
3) being the first human to argue that the CMB birefringence is possibly best explained by birefringent dust crystals
4) doing an exhaustive state of the art overview of past and of all incoming CMB surveys (instruments, balloons, etc) and their consequences, such as e.g. extremely stringent cosmological constraints, a 100% refutation of mainstream inflation, (and in a decade, a direct refutation of LCDM), etc
5) being the first to argue on the viability (within the sensitivity of cheap planned instruments) and the superiority of measuring the CMB temperature drift as an alternative to the redshift drift
6) studying the underappreciated merits of submillimeter KIDS instruments (such as FYST) for characterizing cosmic dust, same for the merits of panchromatic polarimeters such as polstar for the Serkowski Law. And deriving major unreported knowledge, such as the fact that the most accurate radio telescope no longer is ALMA, but is the LMT with its heterodyne receiver.
7) Exhaustively reviewing the known CMB anomalies (axis of evil, odd anisotropy, etc)

**Today I would like to report on a new, fundamental and extremely little known insight regarding the CMB.**
I have previously, extensively argued about the extreme evidence for new physics in the physics of rotation, classifying galaxies by their spin direction basically eliminates the hubble tension, moreover rotation creates an excess redshift which increase with distance. This new source of redshift is the only one to have extensive and diverse empirical evidence in the lab (mossbauer rotors, michelson interferometer reanalysis, the oscillating anomaly, the new rotational doppler effect, and the transverse and sagnac effect, quantized inertia, etc). There is also the topic of rotational universes.
Recently, a new major physical effect has been [proven](https://news.ycombinator.com/item?id=41825906):
The effect, is a novel mechanism to transfer energy from a rotating body to electromagnetic (light) or classical waves.
The effect was predicted a long time ago by Zeldovich, the same researcher that discovered the famous SZ effect (the inprint of clusters on the CMB). However until a few months ago, the effect had never been proven and therefore was never taken into account in cosmological papers!
Firstly, one can observe that everything is rotation in this universe, be it how we model the particle spins, the orbit of electrons arround their atom, to the orbit of planets arround their stars which orbit arrount their galaxy which orbit arround their group/cluster, etc.
At every scales, though for different reasons, things tend to universally spin, relative to themselves and to others. It is the basic unit of orchestration of matter in space, that happen to be stable.
It is this observation that made Gamov ask himself in [1946 ](https://www.nature.com/articles/158549a0) wether we live in a rotating universe (spacetime)

According to my _limited_ understanding, there is a basic and fundamental problem, an omission in standard physics.
We take energy conservation for granted, and also, implicitly, to some extent, fair energy distribution/repartition for granted (CP problem, cosmological principle, etc).
And the basic question is: how can a rotating body in space transfer/convert its rotational (momentum) energy it accumulates, back (e.g. as a radiation)
Well there might be a effect from the [Yarkovsky](https://en.wikipedia.org/wiki/Yarkovsky_effect) and [YORP](https://en.wikipedia.org/wiki/YORP_effect) effects but those are probably too subtle.
There is also [synchrotron radiation](https://en.wikipedia.org/wiki/Synchrotron_radiation) but it probably doesn't suffice or apply in most situations, otherwise what would be the difference with the Zeldovich effect?

The Zeldovich effect (not SZ) state that a rotating body can transfer its energy (momentum) by amplifying (or emitting?) a magnetic wave (light) (which will slow down the object because of energy conservation). **Crucially**, the transfer of energy is only possible if and only if the body rotate faster than the frequency of the magnetic field!
This explains why the effect was never observed before, it require us to make an object rotate at extreme speeds, which is hard to achieve on earth.
Nonetheless, the effect has been proven both for amplifying magnetic waves and [acoustic waves ](https://arxiv.org/pdf/2005.03760)

So you may ask: **What does this have to do with the CMB?**
The new zeldovich effect seems to specifically emit/amplify waves that a rotating material would usually specifically absorb (antenna)
Some people like to brush of the idea of the CMB being generated by cosmic dust because they believe the scientific community understand cosmic dust and know how to model it.
For something that is supposed to behave on known physics, let me tell you that the scientific community has been historically falsified countless times regarding theoretical dust models.
See e.g. the need for the recent astrodust model and its extensions https://arxiv.org/abs/2208.12365
Basically the only surviving dust models require extremely small sizes and some chemical and or topological peculiarities (porous fractals or composite materials, or needles, AKA not dumb carbon/sillicon/iron).

Something almost as little known as the new Zeldovich effect is the fact that among the many CMB "foregrounds/mimickers", there is one that is completely unexpected by standard cosmologists, the so called [AME ](https://arxiv.org/html/2405.04383v1) foreground.
All explanations of the AME foreground have been ruled out, the only one remaining, would be that the AME CMB foreground and polarization would be caused by a widespread population of magnetically aligned micro dust grains (sub millimeter IIRC) spinning at multiple gigahertz!!!

This underappreciated aspect of the universe is absolutely striking, a huge chunk of matter would be present, homogeneously, with an extremely ordered magnetic aligment (ions-atoms too iirc), a strikingly small size (nanometers) and most importantly absolutely mind blowing speeds.
A dust grain rotation at 1GHz means it spin on itself 1 billion times per second!!!!

One might wonder, why not even faster? Is there a speed limit to spinning objects in space? Shouldn't the lack of mechanisms to radiate their momentum energy makes them indefinitely accumulate speed and behave as rotational energy "black holes"?
The CMB AME proves that there is an abundant population of GHZ spinning dust.
The new zeldovich effect is a partial or total answer to the question of wether there is a speed limit to rotating dust.
Cosmic dust made of carbon or cyanogen catalyze microwave absorption/emission, the microwave frequency band is defined as the GHz, therefore, it follows that cosmic dust spinning at GHz speficially amplify/emit microwave radiation!!
Supposedly, the dust grain do not slow down and reach a form of steady state, because their absorption of energy from background radiation (infrared, optical, UV) compensate the energy loss by momentum conversion. As such, cosmic dust acts as a universe wide converter of high frequency light to microwave radiation.

Given the new zeldovich effect, the CMB AME foreground characterization must be recomputed, the missing dust signature too and cosmic dust cmb model must be reimagined.
This is a groundbreaking new physical effect that explains how immense flux of energy in the universe are converted back to light and specifically to microwave, and why there is in practice a speed limit to rotating bodies.
This of course might related witht the afforementioned evidence for new physics in rotation, and especially rotational quantized inertia.

This also allows fast rotating black holes to dissipate their energy (kerr black holes) and probably impact millisecond pulsars
https://en.wikipedia.org/wiki/Millisecond_pulsar
If 1 millisecond is 1 kilohertz then it follows from the zeldovich effect than millisecond pulsars should convert their rotational energy to Kilohertz radiation, which sadly cannot be observed by current radio telescopes.

**Comment by ExpEarth** [comment]
`So I have already extensively written original ideas or knowledge about different aspects of the CMB.`

You've done a pretty good job of bringing all this info.  It's quite a lot now.  I wonder why you don't put some of your work into a paper.  It's hard to keep track of everything you're saying.

`One might wonder, why not even faster? Is there a speed limit to spinning objects in space? Shouldn't the lack of mechanisms to radiate their momentum energy makes them indefinitely accumulate speed and behave as rotational energy "black holes"?
The CMB AME proves that there is an abundant population of GHZ spinning dust.
The new zeldovich effect is a partial or total answer to the question of wether there is a speed limit to rotating dust.`

I don't think your mechanism is good though.  As you have rightly pointed out, we see the CMB temperature "drift" with redshift.  Now the CMB is isotropic at each point in space, even though its temperature changes (here we are talking static universe).  So why should CMB photons moving from all directions impart fast rotations _of a fixed orientation_ to dust grains they encounter?  And why would the CMB photons just cause dust grains to rotate faster and not gas molecules?  The rotations of gas molecules are one of the ways to measure the CMB temperature and so we should see some effect on CMB temperature too in your mechanism.

**Reply by LifeIsStrange** [reply]
> I wonder why you don't put some of your work into a paper.

Thanks. Well I will probably write a paper in my lifetime, besides procrastination, one acute issue I have is that I need someone to endorse me for publishing on Arxiv if I understand correctly.

> why would the CMB photons just cause dust grains to rotate faster and not gas molecules?

You got this backwards. Dust grains becomes naturally extremelly small because of their speed gain. They reach rotational speed that are close to the speed of light because rotational _inertia_ in space means that a rotating object has almost no way to radiate its energy meaning it naturally and extremely slowly gain speed. The energy that allows the gradual speed accumulation comes from absorption of light. Some molecules are more prone to absorb specific light bands. For example cyanogen almost only absorb the CMB light, which is why its temperature is equal to the CMB temperature.
In general though dust will mostly absorb high frequency light (infrared, visible, uv) and reradiate in infrared and microwave.

> why would the CMB photons just cause dust grains to rotate faster and not gas molecules?

Except for e.g. cyanogen, the CMB photons do not cause dust grains to rotate faster. The effect I describe, the Zeldovich effect, says exactly **the opposite, the CMB photons makes dust rotation _slower_.** Dust rotational energy is converted to de novo CMB photons. It is CMB photons because it corresponds to the rotation speed of dust (GHZ) as shown by AME radio observations.


> So why should CMB photons moving from all directions impart fast rotations of a fixed orientation to dust grains they encounter?

As said the CMB slow down dust grains, moreover I don't understand the directionality question, dust rotation is caused by momentum energy gain not angles of incidence, moreover even despite magnetic aligments, dust on cosmological scales is isotropic, just like the CMB.

> The rotations of gas molecules are one of the ways to measure the CMB temperature

I'm not aware of a gas used to measure the CMB temperature, maybe OII?
In principle yes it could be used if it specifically absorbs CMB photons.

> not gas molecules

So your question would actually be does this effect act on molecular scales and if so, should the CMB slow down gas rotation?
It is an excellent question, that I asked myself, is there a size limit to the zeldovich effect? Can molecules or even atoms, spinning at ultra high speeds, give their momentum energy to photons?

I have no idea, nobody has ever addressed this question but in principle there should be no size limit and therefore light should slow down gas molecules and gas molecules should amplify background light. This does not have to be the CMB though as gas probably spin at speeds different from the gigahertz.
related is the propeller effect, also from Zeldovich
https://pmc.ncbi.nlm.nih.gov/articles/PMC3856768/

**Reply by ExpEarth** [reply]
Sorry, but I'm still having trouble visualizing your hypothesis.  Could you possibly put it in the form of a short abstract, as though for a journal article.  What is the problem you are trying to solve and what is your solution?

**Reply by mikehelland** [reply]
>one acute issue I have is that I need someone to endorse me for publishing on Arxiv if I understand correctly.

Getting a pre-print on arxiv is essentially meaningless. If they don't have a journal reference they aren't peer reviewed.  Submit it to a journal, and if they accept it, they'll probably put it on arxiv for you. It's usually part of the publishing process. 

If the un-peer reviewed pre-print still sounds like what you want, you can try https://www.preprints.org/

Having a paper on arxiv before you submit only limits which journals you can submit to, though you're unlikely to run in that. I think there are a very small minority that want exclusive rights too it. Most journals are cool with it. But having a pre-print on arxiv won't help you any in the journey to publication. An un-reviewed paper is an un-reviewed paper, on arxiv or researchgate or a relative's blog or here.

**Comment by LifeIsStrange** [comment]
New paper on cosmic dust and baryonic dark matter by the missing dust signature guy
https://arxiv.org/abs/2411.17775

**Reply by RedshiftDrift** [reply]
Václav Vavryčuk is on the ACG mailing list.  Any questions for him?
